### DISCUSSION TITLE
Cannot render component in swiper #angular

### DISCUSSION THREAD

**Discussion by Kosmonaft** [nolimits4web/swiper#4587] [Q&A]
Hello there!

Before raising an issue wanted to try to solve my problem in a discussion.
I'm trying to move from `ngx-swiper-wrapper` to `swiper` for Angular :-)

Unfortunately the way how my project is setup I do have a lot of components. 
So swipers is made with 2 components for me -> swiper wrapper and swiper slide.
Whenever I want to use a swiper I need to insert the swiper wrapper and then I can do a *ngFor with the swiper slide.

This is not working at all in `swiper` as it seems that `<swiper>` is not handling `<ng-content>` or anything else (even any heading - I guess I need to use the Slots for heading, however I don't need it was more for testing).

Is there a way to use `swiper` in a way that I can create 2 separate components (wrapper and slide)?

Here is a stakblitz with my example https://stackblitz.com/edit/swiper-angular-virtual-tkmhct?file=src/app/s-carousel/s-carousel.component.html

**Comment by vltansky** [comment]
What about this kind of approach: https://stackblitz.com/edit/swiper-angular-virtual-gfwsjg?file=src%2Fapp%2Fs-carousel%2Fs-carousel.component.ts

Although there is "Expression has changed after it was checked" error, so it needs some further work

**Reply by frankfajardo** [reply]
Unfortunately, besides the console error you identified, it does not appear to work when `loop` is `true`. Do you see another approach? Thanks heaps in advance
